src/libraries/System.Net.Http.Json/src/System/Net/Http/Json/JsonContent.cs

```diff
@@ -20,8 +20,8 @@ public sealed partial class JsonContent : HttpContent
         private static MediaTypeHeaderValue DefaultMediaType
```

Comment:
@Jozkee Could we squeeze a bit more performance out of it by also switching to a field and reusing the default media type instance instead of having the lambda?
```csharp
        private static readonly MediaTypeHeaderValue s_defaultMediaType
            = new MediaTypeHeaderValue(JsonMediaType) { CharSet = "utf-8" };
```

Reply:
I did consider that as well, but as per @scalablecory's, we must not re-use this, it could be modified or disposed on a request, or it can be intercepted by a `DelegatingHandler`.